gpflow/kernels/periodic.py

```diff
@@ -31,26 +31,23 @@ class Periodic(Kernel):
     σ² is the variance parameter,
     γ is the period parameter.
 
-    (note that usually we have a factor of 4 instead of 0.5 in front but this
-    is absorbed into the lengthscale hyperparameter).
+    NOTE: that usually we have a factor of 4 instead of 0.5 in front but this
+        is absorbed into the lengthscale hyperparameter.
+    NOTE: periodic kernel uses `active_dims` of a base kernel, therefore
+        the constructor doesn't have it as an argument.
```

Comment:
I don't think you need this last note

Reply:
We do, users were confused (see #1230)

Reply:
I kind of agree that this line is not necessary. The signature of the constructor is self-explanatory. Unfortunately, https://github.com/GPflow/GPflow/issues/1230 doesn't have many details about what was wrong with the documentation. However, I prefer to keep this mention to make it crystal clear that a periodic kernel uses active_dims from a base kernel.

Reply:
ok. happy to leave as is